docs/src/main/asciidoc/security-architecture-concept.adoc

```diff
@@ -23,19 +39,27 @@ In other contexts, it is possible to have other parallel representations of the
 
 For more information, see xref:security-identity-providers-concept.adoc[Identity providers].
 
+=== `SecurityIdentityAugmentor`
+Because Quarkus Security is customizable, for example, you can add authorization roles to `SecurityIdentity`, you can register and prioritize one or more custom security augmentors.
+
+Registered instances of `SecurityIdentityAugmentor` are invoked during the final stage of the security authentication process. `IdentityProvider` passes `SecurityIdentity` to `SecurityIdentityAugmentor` to customize `SecurityIdentity`.
```

Comment:
I propose to drop `IdentityProvider passes SecurityIdentity to SecurityIdentityAugmentor to customize SecurityIdentity.` - at the diagram is looks as if it is indeed `IdentityProvider` which passes the identity to the augmentor, but it is done as part of the higher-level orchestration by the security manager, once the authentication mechanism and identity provider pair complete their work on creating the initial identity

Reply:
Fixed. Sentence removed.  :+1: